apps/google-analytics-4/frontend/src/hooks/useGetFieldValue.tsx

```diff
@@ -0,0 +1,16 @@
+import { useFieldValue } from '@contentful/react-apps-toolkit';
+
+const useGetFieldValue = (fieldId: string) => {
```

Comment:
I don't think this needs to be a hook actually. Can't you just make this a regular `.ts` file since there's no XML comopnents here. This means you can just make this a normal function that's exported.

One key pattern of a hook is that it almost always has an useEffect inside the hook itself. If it doesn't have an useEffect, then it either has some other hook like memo, callback, or reducer etc...

Reply:
@ryunsong-contentful and I synced about this -- since the `useGetFieldValue` hook is calling `useFieldValue`, this needs to be in a custom hook or else we see the error `React Hooks must be called in a React function component or a custom React Hook function`.